include/analytics.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/false
+
+# This is the server that is collecting zopen usage stats
+ZOPEN_STATS_URL="http://163.74.88.212:3000"
+
+isAnalyticsOn()
+{
+  jsonConfig="${ZOPEN_ROOTFS}/etc/zopen/config.json"
+  if [ ! -f ${jsonConfig} ]; then
+    printError "config.json file does not exist. This should not occur. Please report an issue"
+  fi
+  isCollecting=$(jq -r '.is_collecting_stats' $jsonConfig)
+  if [ "$isCollecting" = "true" ] || [ -z "${ZOPEN_ANALYTICS_JSON}" ]; then
+    return 0
+  else
+    return 1
+  fi
+}
+
+isIBMHostname()
+{
+  ip_address=$(/bin/dig +short "$(hostname)" | tail -1)
```

Comment:
is /bin/dig in z/OS 2.4 and up? If so, all good.

Reply:
yep, `/bin/dig` is present in all the machines I've tested so far (2.4 and up)